src/sparql/keywords.ts

```diff
@@ -1,626 +1,152 @@
+  LANGMATCHES: createKeyword({ name: 'LANGMATCHES' }),
+  DATATYPE: createKeyword({ name: 'DATATYPE' }),
+  BOUND: createKeyword({ name: 'BOUND' }),
+  IRI: createKeyword({ name: 'IRI' }),
+  URI: createKeyword({ name: 'URI' }),
+  BNODE: createKeyword({ name: 'BNODE' }),
+  RAND: createKeyword({ name: 'RAND' }),
+  ABS: createKeyword({ name: 'ABS' }),
+  CEIL: createKeyword({ name: 'CEIL' }),
+  FLOOR: createKeyword({ name: 'FLOOR' }),
+  ROUND: createKeyword({ name: 'ROUND' }),
+  CONCAT: createKeyword({ name: 'CONCAT' }),
+  STRLEN: createKeyword({ name: 'STRLEN' }),
+  UCASE: createKeyword({ name: 'UCASE' }),
+  LCASE: createKeyword({ name: 'LCASE' }),
+  ENCODE_FOR_URI: createKeyword({ name: 'ENCODE_FOR_URI' }),
+  CONTAINS: createKeyword({ name: 'CONTAINS' }),
+  STRSTARTS: createKeyword({ name: 'STRSTARTS' }),
+  STRENDS: createKeyword({ name: 'STRENDS' }),
+  STRBEFORE: createKeyword({ name: 'STRBEFORE' }),
+  STRAFTER: createKeyword({ name: 'STRAFTER' }),
+  YEAR: createKeyword({ name: 'YEAR' }),
+  MONTH: createKeyword({ name: 'MONTH' }),
+  DAY: createKeyword({ name: 'DAY' }),
+  HOURS: createKeyword({ name: 'HOURS' }),
+  MINUTES: createKeyword({ name: 'MINUTES' }),
+  SECONDS: createKeyword({ name: 'SECONDS' }),
+  TIMEZONE: createKeyword({ name: 'TIMEZONE' }),
+  TZ: createKeyword({ name: 'TZ' }),
+  NOW: createKeyword({ name: 'NOW' }),
+  UUID: createKeyword({ name: 'UUID' }),
+  STRUUID: createKeyword({ name: 'STRUUID' }),
+  MD5: createKeyword({ name: 'MD5' }),
+  SHA1: createKeyword({ name: 'SHA1' }),
+  SHA256: createKeyword({ name: 'SHA256' }),
+  SHA384: createKeyword({ name: 'SHA384' }),
+  SHA512: createKeyword({ name: 'SHA512' }),
+  COALESCE: createKeyword({ name: 'COALESCE' }),
+  IF: createKeyword({ name: 'IF' }),
+  STRLANG: createKeyword({ name: 'STRLANG' }),
+  STRDT: createKeyword({ name: 'STRDT' }),
+  sameTerm: createKeyword({ name: 'sameTerm' }),
+  isIRI: createKeyword({ name: 'isIRI' }),
+  isURI: createKeyword({ name: 'isURI' }),
+  isBLANK: createKeyword({ name: 'isBLANK' }),
+  isLITERAL: createKeyword({ name: 'isLITERAL' }),
+  isNUMERIC: createKeyword({ name: 'isNUMERIC' }),
+  REGEX: createKeyword({ name: 'REGEX' }),
+  SUBSTR: createKeyword({ name: 'SUBSTR' }),
+  REPLACE: createKeyword({ name: 'REPLACE' }),
+  EXISTS: createKeyword({ name: 'EXISTS' }),
+  NOT_EXISTS: createKeyword({ name: 'NOT_EXISTS', pattern: /NOT EXISTS/i }),
+  COUNT: createKeyword({ name: 'COUNT' }),
+  SUM: createKeyword({ name: 'SUM' }),
+  MIN: createKeyword({ name: 'MIN' }),
+  AVG: createKeyword({ name: 'AVG' }),
+  SAMPLE: createKeyword({ name: 'SAMPLE' }),
+  GROUP_CONCAT: createKeyword({ name: 'GROUP_CONCAT' }),
+  SEPARATOR: createKeyword({ name: 'SEPARATOR' }),
+  TRUE: createKeyword({ name: 'TRUE' }),
+  FALSE: createKeyword({ name: 'FALSE' }),
+  IN: createKeyword({ name: 'IN' }),
+  NOT_IN: createKeyword({ name: 'NOT_IN', pattern: /NOT IN/i }),
+  MAX: createKeyword({ name: 'MAX', longer_alt: MAX_LENGTH }),
```

Comment:
On second thought, I think this should also use the `unknown` longer_alt. We can use the token ordering to match `max_length` first, then use the `unknown` longer_alt for `max` (ie. `maximum`), then `max` itself. This would mean that `MAX_LENGTH`, currently defined at the top of the file, can also be defined in line like the rest of the keywords.